sormas-backend/src/main/java/de/symeda/sormas/backend/event/Event.java

```diff
@@ -629,4 +631,13 @@ public void setSormasToSormasOriginInfo(SormasToSormasOriginInfo originInfo) {
 	public void setSormasToSormasShares(List<SormasToSormasShareInfo> sormasToSormasShares) {
 		this.sormasToSormasShares = sormasToSormasShares;
 	}
+
+	@Column(columnDefinition = "text")
```

Comment:
The ID in SurvNet (`NameGA`) seems to be limited to a 255 characters string. Having the internalId as a "text" data type in SORMAS might end up in issues when sending events to SurvNet.

Reply:
I'm not sure this needs to be handled in SORMAS, I would say it's the responsibility of the converter to handle this kind of issues.

In SORMAS we have varchar(255) only for enums, for free text we have varchar(512) or text.

@MateStrysewske what do you think, should the size of this field be limited to 255 in SORMAS?

Reply:
@leventegal-she No, I don't think so. I guess the danger of someone entering a value here that is too long is pretty low anyway. Moreover, if this limitation changes at any point in the future, we'll have troubles with removing it in the mobile app, so I'd rather have it as a text right from the start.